src/pages/home/report/ReportActionCompose.js

```diff
@@ -324,6 +333,27 @@ class ReportActionCompose extends React.Component {
                                                 animationIn="fadeInUp"
                                                 animationOut="fadeOutDown"
                                                 menuItems={[
+                                                    ...(Permissions.canUseIOU() ? [
+                                                        hasMultipleParticipants
+                                                            ? {
+                                                                icon: Receipt,
+                                                                text: 'Split Bill',
+                                                                onSelected: () => {
+                                                                    Navigation.navigate(
+                                                                        ROUTES.getIouSplitRoute(this.props.reportID),
+                                                                    );
+                                                                },
+                                                            }
+                                                            : {
+                                                                icon: MoneyCircle,
+                                                                text: 'Request Money',
+                                                                onSelected: () => {
+                                                                    Navigation.navigate(
+                                                                        ROUTES.getIouRequestRoute(this.props.reportID),
+                                                                    );
+                                                                },
+                                                            },
+                                                    ] : []),
```

Comment:
Interesting way to add additional items without needing logic defined outside of the prop. Nice!